addons/calendar/models/res_users.py

```diff
@@ -176,3 +176,7 @@ def check_calendar_credentials(self):
 
     def check_synchronization_status(self):
         return {}
+
+    def _has_any_active_synchronization(self):
+        """ Overridable method for checking if user has any synchronization active in inherited modules. """
```

Comment:
We can add here the output type as done in the inherited modules
```suggestion
        """
            Overridable method for checking if user has any synchronization active in inherited modules. 
            :return: boolean indicating if any synchronization is active.
        """
```